docusearch/models.py

```diff
@@ -28,7 +28,14 @@ class Document(models.Model):
     text = models.TextField(
         null=False, help_text='The full text of the document')
     title = models.TextField(null=True)
-    date = models.DateField(null=True)
+    date_created = models.DateField(
```

Comment:
This is exactly how I made the date changes in an earlier branch. I'll fix mine to match yours I think. I called them 'create_date' and 'release_date' but that doesn't matter as much.